src/sidebar/services/annotations-exporter.ts

```diff
@@ -49,22 +56,25 @@ export class AnnotationsExporter {
 
   buildTextExportContent(
     annotations: APIAnnotationData[],
-    groupName = '',
-    /* istanbul ignore next - test seam */
-    now = new Date(),
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: TextExportOptions,
   ): string {
     const [firstAnnotation] = annotations;
     if (!firstAnnotation) {
       throw new Error('No annotations to export');
     }
 
+    const extractUserName = (annotation: APIAnnotationData) =>
```

Comment:
```suggestion
    const extractUsername = (annotation: APIAnnotationData) =>
```

By convention we treat "username" as one word.

Reply:
Haha, I thought about this, but I wanted to express the intention of "extracting the user's name", which can be the display name, or the username. Hence, the capital `N`.

But I think it's more confusing. "Extract username" is probably clear enough"

Reply:
I see what you mean. That makes sense actually. Feel free to leave as is.